Hymns/Infrastructure/Dependency Injection/AppDelegate+Injection.swift

```diff
@@ -23,7 +23,8 @@ extension Resolver: ResolverRegistering {
         register {URLSession.shared}.scope(application)
         register {AnalyticsLogger()}
         register {SystemUtil()}
-        register {PDFLoader()}
+        register {PDFLoaderImpl()}
```

Comment:
You don't need this if you have registerPDFLoader()

Reply:
Addressed in recent commit